README.md

```diff
@@ -110,12 +110,12 @@ __Help! Things are not working!__
 [Eclipse]: https://eclipse.org/
 [Gradle]: https://gradle.org/
 [Homepage]: https://spongepowered.org/
-[IntelliJ]: http://www.jetbrains.com/idea/
+[IntelliJ]: https://www.jetbrains.com/idea/
 [Issues]: https://github.com/SpongePowered/Sponge/issues
 [Documentation]: https://docs.spongepowered.org/
-[Java]: http://java.oracle.com/
+[Java]: https://java.oracle.com/
 [Source]: https://github.com/SpongePowered/Sponge/
-[MIT License]: http://www.tldrlegal.com/license/mit-license
+[MIT License]: https://www.tldrlegal.com/license/mit-license
```

Comment:
Should be changed to:
`[MIT License]: https://tldrlegal.com/license/mit-license`